src/main/java/ch/njol/skript/expressions/ExprDrops.java

```diff
@@ -51,24 +52,33 @@
  * @author Peter Güttinger
  */
 @Name("Drops")
-@Description("Only works in death events. Holds the drops of the dying creature. Drops can be prevented by removing them with \"remove ... from drops\", e.g. \"remove all pickaxes from the drops\", or \"clear drops\" if you don't want any drops at all.")
+@Description("Only works in death events (Also block break events in 1.12+). Holds the drops of the dying creature. Drops can be prevented by removing them with \"remove ... from drops\", e.g. \"remove all pickaxes from the drops\", or \"clear drops\" if you don't want any drops at all.")
```

Comment:
Consider rewriting the description instead of just adding in middle of it. Right now, it is confusing and grammatically incorrect.

Reply:
Okay I re-wrote it, hopefully its better.
If anyone does not like this please let me know and/or give feedback/input on what you would like to see here.